programmers/20210503/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+    for(var i = 0; i < seoul.length; i++){
+        if (seoul[i] == "Kim"){
+            answer = i;
+        }
+    }
```

Comment:
변수를 미리 선언하지 않는다면 var answer가 되므로 let을 이용해서 미리 선언해주는 것이 좋습니다.

```suggestion
function solution(seoul) {
    let answer = "";
    
    for (let i = 0; i < seoul.length; i++){
        if (seoul[i] === "Kim"){
            answer = i;
        }
    }
```